src/libs/bridge/ReactNativeLauncherMessenger.js

```diff
@@ -46,7 +46,7 @@ export default class ReactNativeLauncherMessenger extends MessengerInterface {
   onMessage(event) {
     const data = JSON.parse(event.nativeEvent.data)
     if (this.listener) this.listener({ data })
-    if (this.debug && data.type === '@post-me') {
+    if (this.debug && data.type === '@post-me' && data.eventName !== 'log') {
```

Comment:
This commit should be squashed with the one adding the `post-me` check. But I still don't understand the need.

Reply:
These are different : 
 * this one hides post-me message related to the transmission of console.logs in the webviews
 * the one adding post-me check hides postMessage messages not related to post-me

I don't think we should merge them.

Globally, the need here is to make log messages related to post-me easier to scan visually and easier to link, making it easier to find messages related to a given session or a given request.